MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -930,7 +930,7 @@ private void BuildStreamVideoItem(
 
             var appliedAudioConditions = options.Profile.CodecProfiles
                 .Where(i => i.Type == CodecType.VideoAudio &&
-                    i.ContainsAnyCodec(audioStream?.Codec, container) &&
+                    i.ContainsAnyCodec(audioCodec, container) &&
```

Comment:
Reviewing #9374, now I think we probably need to use `playlistItem.AudioCodecs` in place of `audioCodec` because the latter remains unchanged - without HLS restrictions (https://github.com/nyanmisaka/jellyfin/blob/7bd971ffefe983a1f32836b0f2fcbe602f92efa0/MediaBrowser.Model/Dlna/StreamBuilder.cs#L843-L853).

Reply:
Also, here https://github.com/jellyfin/jellyfin/blob/2fe13f54eaf87eefefd27f4ccb2ace1371f5e886/MediaBrowser.Model/Dlna/StreamBuilder.cs#L940

Or maybe just assign it back?
```cs
audioCodec = playlistItem.AudioCodecs;
```